src/client/routes/Vaults/index.tsx

```diff
@@ -223,54 +215,58 @@ export const Vaults = () => {
 
   return (
     <ViewContainer>
-      <SummaryCard header={t('dashboard.header')} items={summaryCardItems} variant="secondary" cardSize="small" />
+      <StyledSliderCard
+        header={t('vaults:your-time-card.header')}
+        Component={
+          <Text>
+            <p>{t('vaults:your-time-card.desc-1')}</p>
+            <p>{t('vaults:your-time-card.desc-2')}</p>
+            <p>{t('vaults:your-time-card.desc-3')}</p>
+          </Text>
+        }
+        // slidesContent={[
+        // {
+        //   header: 'Test header',
+        //   content: 'test content',
+        // },
+        // ]}
```

Comment:
Remove comments

Reply:
Added comment, just leave it so we have an easier implementation when we get more data to slide show